starter-code/index.js

```diff
@@ -1,11 +1,42 @@
 class SortedList {
-  constructor() {}
-  add(item) {}
-  get(pos) {}
-  max() {}
-  min() {}
-  average() {}
-  sum() {}
+  constructor() {
+    this.length = 0;
+    this.items = [];
+  }
+  add(item) {
+    this.length += 1;
+    this.items.push(item)
+    this.items.sort((a, b) => a - b);
+  }
+  get(pos) {
+    let index = pos - 1;
+    return this.items[index];
```

Comment:
- está bien que crees una variable para la pos-1, pero igual no es necesario, this.items[pos - 1].
- también te ha faltado comprobar si la pos existe en el array, y si no, lanzar un error "OutofBounds"